src/wamp-server-over-fs.ts

```diff
@@ -1,12 +1,16 @@
 import * as Promise from 'bluebird';
 import {Connection, Session} from 'autobahn';
 import {FileSystem, fileSystemEventNames, fileSystemMethods} from './api';
-
 const Router = require('wamp-server');
 
+
+export class WampConnection extends Connection {
+    isConnected: boolean
```

Comment:
@amir-arad  I've told that autobahn typings are old, and don't contain that field